core/src/test/scala/caliban/wrappers/WrappersSpec.scala

```diff
@@ -272,51 +268,6 @@ object WrappersSpec extends ZIOSpecDefault {
           }
         )
       },
-      test("Apollo Caching") {
```

Comment:
I won't revert this one, just to make it easier to delete the file in the next major release